Sources/StreamChatUI/CommonViews/ChatInputView/ChatCommandLabel.swift

```diff
@@ -52,25 +51,27 @@ open class _ChatMessageInputSlashCommandView<ExtraData: ExtraDataTypes>: _View,
         commandLabel.adjustsFontForContentSizeCategory = true
         commandLabel.textAlignment = .center
         
-        iconView.image = uiConfig.images.messageComposerCommand.tinted(with: uiConfig.colorPalette.staticColorText)
+        iconView.image = uiConfig.images.messageComposerCommand
+            .tinted(with: uiConfig.colorPalette.staticColorText)
     }
     
     override open func setUpLayout() {
+        super.setUpLayout()
+
         embed(container)
-        
         container.preservesSuperviewLayoutMargins = true
         container.isLayoutMarginsRelativeArrangement = true
 
         container.addArrangedSubview(iconView)
-        iconView.isHidden = false
-
         container.addArrangedSubview(commandLabel)
+        iconView.isHidden = false
         commandLabel.isHidden = false
         
         iconView.contentMode = .scaleAspectFit
+        layer.cornerRadius = bounds.height / 2
     }
     
     override open func updateContent() {
-        commandLabel.text = commandName
+        commandLabel.text = content?.name.uppercased()
```

Comment:
Should we always invoke `super` in case we add smth there in the future?

Reply:
I'm actually not sure 🤔 But now the pattern is that we should avoid at all costs to subclass views, and only use composition. But I think all views are calling super, so to follow the same pattern, I'll do the same, thanks 👍